controllers/form.go

```diff
@@ -127,3 +129,59 @@ var GetForm = func(w http.ResponseWriter, r *http.Request) {
 
 	u.Respond(w, form, 200)
 }
+
+var GetAllForms = func(w http.ResponseWriter, r *http.Request) {
+	// Get roll number
+	rno := GetRollNo(w, r, false)
+
+	// To extrach data out of collection.Find() 
+	type formDB struct {
+		ID		primitive.ObjectID		`bson:"_id"`
+		Name 	string					`bson:"name"`
+		Token 	string					`bson:"responsetoken"`
+	}
+
+	// To send data to frontend	
+	type formDetails struct {
+		ID		string	
+		Name 	string
+		Token 	string
+	}
+
+	var forms []formDetails
+
+	collection := u.Collection("forms")
+	count, err := collection.CountDocuments(u.Context(), bson.M{"creator": rno})
+	if count == 0 {
+		u.Respond(w, forms, 200)
+	}
+
+	// To Set which fields are required in the output
+	type fields struct {
+		ID		int		`bson:"_id"`
+		Name 	int		`bson:"name"`
+		Token 	int		`bson:"responsetoken"`
```

Comment:
This indentation is strange. Use the latest VS Code with the official golang extension and install the required modules to ensure this doesn't happen.

Reply:
Fixed, thanks for pointing out.